### PR TITLE
[AMS-298] templates and advertisements migration to whitelist

### PR DESCRIPTION
**Pull request by xakep139**

### CODE REVIEW

File: src/MigrationTool/ApiRepository.cs [hunk 1]
```diff
@@ -383,8 +367,7 @@ public ApiRepository(ILogger<ApiRepository> logger, Uri apiUri, Uri storageUri,
 
                     using (var response = await _httpClient.PostAsync(url, content))
                     {
-                        stringResponse = await response.Content.ReadAsStringAsync();
-                        _logger.LogDebug(_logFormat, response.RequestMessage.Method, url, string.Empty, string.Empty, stringResponse);
+                        stringResponse = (await HandleResponse(response)).response;
```
Comment (denisivan0v) [inline src/MigrationTool/ApiRepository.cs]: ambiguous naming here for the `response` term

File: src/MigrationTool/ApiRepository.cs [hunk 2]
```diff
@@ -481,5 +497,21 @@ public ApiRepository(ILogger<ApiRepository> logger, Uri apiUri, Uri storageUri,
                 throw new WebException("Can't establish connection with API");
             }
         }
+
+        private async Task<(string response, string server, string requestId)> HandleResponse(HttpResponseMessage response)
```
Comment (denisivan0v) [inline src/MigrationTool/ApiRepository.cs]: Let's capitalize tuple value names to fix ambiguous naming

File: src/MigrationTool/ApiRepository.cs (hunk 1 above)
Comment (xakep139) [inline src/MigrationTool/ApiRepository.cs]: Done in https://github.com/2gis/nuclear-vstore/pull/52/commits/fc98c3c6f4480e15941074861729f8372b2775fe

File: src/MigrationTool/ApiRepository.cs (hunk 2 above)
Comment (xakep139) [inline src/MigrationTool/ApiRepository.cs]: Done in https://github.com/2gis/nuclear-vstore/pull/52/commits/fc98c3c6f4480e15941074861729f8372b2775fe

**Pull request merged**